app/components/toast-notification/styles.js

```diff
@@ -34,6 +34,12 @@ export default StyleSheet.create({
   successText: {
     color: Theme.fontColors.white
   },
+  warn: {
```

Comment:
Lastly, I think this color shouldn't be called `warn`. I think of yellow when I read `warn`, maybe `disabled`?